automated_testing/defaults/run_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+echo "You should really fix this, you know?"
```

Comment:
We probably should :joy_cat:    Is this intended as a placeholder for tests that don't do the right thing? Maybe some verbage indicating that `run_test.sh` for the specific test case needs to be implemented?

Reply:
It looks like this is a test case template. One would run `new_testcase my_test_case` which creates a new directory named `my_test_case` and copy this file and all others in the `defaults` directory to that new directory. So it makes sense that a template for a new test would have a failing test.